### DISCUSSION TITLE
How do I hide a form from being displayed in taskbar?

### DISCUSSION THREAD

**Discussion by Shadowblitz16** [picoe/Eto#2156] [Q&A] [answered]
I am trying to create a mdi hack where I just let the windows float freely on the desktop instead of being contained by a mdi client rectangle, However I need to hide the associated forms with the main form from the taskbar.

Is there a way to do this? 
Basicly I want to hide the Common.Gtk window from the task bar.
![image](https://user-images.githubusercontent.com/2453707/156486805-e98d06c6-e137-4683-b94f-241ba130f5ad.png)

**Comment by cwensley** [comment, marked as answer]
Does `Window.ShowInTaskbar` not work?  If not, it could be a bug.

**Reply by Shadowblitz16** [reply]
This does work thankyou! I just wasn't sure what the method or property was called.
